rfcs/20210403-DMF-v1.0.0.md

```diff
@@ -0,0 +1,122 @@
+---|---|---|---|---|---
+Joe Shmo | lead | jshmo | jshmo | jshmo@nanocool.co | active
+Alice Land | developer | alice1 | alice2 | alice@wonderland | inactive
+
+**nanoMFG Github Team(s):** @coolteam
+**nanoHUB Group(s):**
+
+## 1. Introduction
+
+
+### 1.1 Purpose and Vision Statement
+<!-- Why are we building this tool?
+What is the key benefit
+How does it relate to existing tools and existing software?
+How does it fit into the overall objectives for the nano **manufacturing** node?
+Who will use this software?
+-->
+The first version of the DMf is intended to provide experimental labs working with 2D materials with framework to describe, catalog, analyse and publish data.
+
+##### Relationship to Existing Software
+Seems there are a number of commertional options for scientific data management software (SDMS)
+
+##### Our Approach
+
+### 1.2 References
+<!--List any documents or background material that are relevant.  Links are useful. For instance, a link to a wiki or readme page in the project repository, or link to a uploaded file (doc, pdf, ppt, etc.).-->
+
+## 2 Overview and Major Planned Features
+<!--Provide and overview characterising this proposed release.  Describe how users will interact with each proposed feature. Include a schematic/diagram to illustrate an overview of proposed software and achitecture componets for the project-->
+
+### 2.1 Product Background and Strategic Fit
+<!--Provide context for the proposed product.  Is this a completely new projects, or next version of an existing project? This can include a description of any contextual research, or the status of any existing prototype application.  If this SPD describes a component, describe its relationship to larger system. Can include diagrams.-->
+
+### 2.2 Scope and Limitations for Current Release
+<!--List the all planned goals/features for this release.  These should be links to issues.  Add a new subsection for each release.  Equally important, document feature you explicity are not doing at this time-->
+
+
+##### 2.2.1 Planned Features
+
+#### 2.2.2 Release Notes 
+##### v#.#.#
+
+### 2.3 Scope and Limitations for Subsequent Releases
+<!--Short summary of  future envisioned roadmap for subsequent efforts.-->
+
+### 2.3 Operating Environment
+<!--Describe the target environment.  Identify components or application that are needed.  Describe technical infrastructure need to support the application.-->
+
+### 2.4 Design and Implementation Constraints
+<!--This could include pre-existing code that needs to be incorporated ,a certain programming language or toolkit and software dependencies.  Describe the origin and rationale for each constraint.-->
+
+## 3 User Interaction and Design
+
+### 3.1 Classes of Users
+<!--Identify classes (types) of users that you anticipate will use the product.  Provide any relevant context about each class that may influence how the product is used: 
+The tasks the class of users will perform
```

Comment:
maybe also how they do those tasks now, if they do, and how this will be an improvement?